bin/grouper-api

```diff
@@ -29,6 +30,34 @@ class Application(tornado.web.Application):
         super(Application, self).__init__(*args, **kwargs)
 
 
+class SendEmailThread(Thread):
+    def run(self):
+        while True:
+            logging.debug("Sending emails...")
+            try:
+                now = datetime.utcnow()
+                session = Session()
+                emails = session.query(AsyncNotification).filter(
+                    AsyncNotification.sent == False).all()
```

Comment:
Yeah, I'm not sure how that would avoid the race. They won't error -- it's perfectly valid to update a column to the same value twice?

Anyway, I was being lazy, I can update this. There's a method of doing this I think that will work better, it just requires a bunch of back and forths with the db.